### ISSUE TITLE
[JENKINS-54304] Jobs in parallel dont display the variant they are running.

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/workflow-job-plugin#673]
<p>This issue manifested itself since October 15th.</p>

<p>When a build is triggered and it runs on multiple slaves parallely, you normally get an output for each line corresponding to the variant its running. Since a Jenkins update on October 15th, this has not been the case. This is making it harder for us to debug the issues on the variants/slaves, as we don't know on which variant the issue is being manifested. </p>

<p><em>NOTE:</em></p>

<p>Although unrelated, this issue was manifested together with another bug which doesn't allow an anonymous user to view jobs, even if logged in as a non-admin. Only users with admin privileges are allowed to view the logs. That issue has been found on your backlog.  <a href="https://issues.jenkins-ci.org/browse/JENKINS-54031" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/JENKINS-54031</a></p>

<p> </p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=fhajredini">fhajredini</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304" target="_blank">Jobs in parallel dont display the variant they are running.</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jglick.png" title="jglick's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jglick">jglick</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>component(s)</b>: workflow-job-plugin
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2019-08-14T22:30:10+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 16
<li><b>imported</b>: 20251212-090250
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>This issue manifested itself since October 15th.</p>

<p>When a build is triggered and it runs on multiple slaves parallely, you normally get an output for each line corresponding to the variant its running. Since a Jenkins update on October 15th, this has not been the case. This is making it harder for us to debug the issues on the variants/slaves, as we don't know on which variant the issue is being manifested. </p>

<p><em>NOTE:</em></p>

<p>Although unrelated, this issue was manifested together with another bug which doesn't allow an anonymous user to view jobs, even if logged in as a non-admin. Only users with admin privileges are allowed to view the logs. That issue has been found on your backlog.  <a href="https://issues.jenkins-ci.org/browse/JENKINS-54031" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/JENKINS-54031</a></p>

<p> </p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>Jenkins 2.138.2</code></li></ul>
<details><summary><i>20 attachments</i></summary>

- [JENKINS-54304.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45435/JENKINS-54304.png)
  > ![JENKINS-54304.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45435/JENKINS-54304.png)
- [NewNodeConsoleNote-script.js](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45147/NewNodeConsoleNote-script.js)
- [Selection_047.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/44884/Selection_047.png)
  > ![Selection_047.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/44884/Selection_047.png)
- [Selection_048.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/44883/Selection_048.png)
  > ![Selection_048.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/44883/Selection_048.png)
- [earlier.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/46673/earlier.JPG)
  > ![earlier.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/46673/earlier.JPG)
- [earlier.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/46668/earlier.JPG)
  > ![earlier.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/46668/earlier.JPG)
- [image-2018-11-06-14-35-41-221.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45072/image-2018-11-06-14-35-41-221.png)
  > ![image-2018-11-06-14-35-41-221.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45072/image-2018-11-06-14-35-41-221.png)
- [image-2018-11-06-14-39-16-393.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45073/image-2018-11-06-14-39-16-393.png)
  > ![image-2018-11-06-14-39-16-393.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45073/image-2018-11-06-14-39-16-393.png)
- [image-2018-11-08-15-38-49-986.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45096/image-2018-11-08-15-38-49-986.png)
  > ![image-2018-11-08-15-38-49-986.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45096/image-2018-11-08-15-38-49-986.png)
- [image-2018-11-08-15-39-26-889.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45097/image-2018-11-08-15-39-26-889.png)
  > ![image-2018-11-08-15-39-26-889.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45097/image-2018-11-08-15-39-26-889.png)
- [image-2018-11-08-15-41-10-158.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45098/image-2018-11-08-15-41-10-158.png)
  > ![image-2018-11-08-15-41-10-158.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45098/image-2018-11-08-15-41-10-158.png)
- [image-2018-11-20-16-58-04-034.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45226/image-2018-11-20-16-58-04-034.png)
  > ![image-2018-11-20-16-58-04-034.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45226/image-2018-11-20-16-58-04-034.png)
- [image-2018-11-20-17-20-19-040.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45228/image-2018-11-20-17-20-19-040.png)
  > ![image-2018-11-20-17-20-19-040.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45228/image-2018-11-20-17-20-19-040.png)
- [jenkins-browser-log](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45146/jenkins-browser-log)
- [log](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45124/log)
- [log-index](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45125/log-index)
- [now.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/46672/now.JPG)
  > ![now.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/46672/now.JPG)
- [now.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/46670/now.JPG)
  > ![now.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/46670/now.JPG)
- [now.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/46671/now.JPG)
  > ![now.JPG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/46671/now.JPG)
- [two-variants.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45123/two-variants.png)
  > ![two-variants.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45123/two-variants.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>fhajredini:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=352988&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-352988">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>That did not fix the issue we are having. Everything seems to be running the same way.</p>
</pre>
</details>

</ul>
</details>

<p>That did not fix the issue we are having. Everything seems to be running the same way.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jglick.png" title="jglick's avatar" /> jglick:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=353080&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-353080">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Working for me… make sure you are actually running the stated build (check <b>Plugin Manager » Installed</b>) and attach a screenshot.</p>
</pre>
</details>

</ul>
</details>

<p>Working for me… make sure you are actually running the stated build (check <b>Plugin Manager » Installed</b>) and attach a screenshot.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>fhajredini:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=353131&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-353131">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Here you go: </p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/45072/45072_image-2018-11-06-14-35-41-221.png" style="border: 0px solid black" /></span></p>

<p>I cant find the specific version ID, but here is another one which asks me to downgrade it to the previous one:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/45073/45073_image-2018-11-06-14-39-16-393.png" style="border: 0px solid black" /></span></p>
</pre>
</details>

</ul>
</details>

<p>Here you go: </p>

<p><span class="image-wrap" style=""><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/45072/image-2018-11-06-14-35-41-221.png" style="border: 0px solid black" /></span></p>

<p>I cant find the specific version ID, but here is another one which asks me to downgrade it to the previous one:</p>

<p><span class="image-wrap" style=""><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45073/image-2018-11-06-14-39-16-393.png" style="border: 0px solid black" /></span></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>fhajredini:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=354282&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-354282">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>-Unzipped workflow-job.jpi 
--unzipped WEB-INF/lib/workflow-job.jar
&#8212; Checked WEB-INF/lib/org/jenkinsci/plugins/workflow/job/console/NewNodeConsoleNote/script.js. The File checks out with the latest github code.</p>

<p>First time the variants werent visible, so i force refreshed my browser. I got them (although slightly disoriented)
<span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/45228/45228_image-2018-11-20-17-20-19-040.png" style="border: 0px solid black" /></span></p>

<p> </p>

<p>Thank you for the help.</p>
</pre>
</details>

</ul>
</details>

<p>-Unzipped workflow-job.jpi <br/>
--unzipped WEB-INF/lib/workflow-job.jar<br/>
&#8212; Checked WEB-INF/lib/org/jenkinsci/plugins/workflow/job/console/NewNodeConsoleNote/script.js. The File checks out with the latest github code.</p>

<p>First time the variants werent visible, so i force refreshed my browser. I got them (although slightly disoriented)<br/>
<span class="image-wrap" style=""><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45228/image-2018-11-20-17-20-19-040.png" style="border: 0px solid black" /></span></p>

<p> </p>

<p>Thank you for the help.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>fhajredini:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=354433&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-354433">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>The issue doesnt seem that minor anymore. The  <span class="error">&#91;variant&#93;</span> is not searchable through CTRL+F. Also if u view the logs as plain text, you get no information of the variants. Having ~10MB of logs with &gt;5 different variants makes it difficult to track the variant.</p>
</pre>
</details>

</ul>
</details>

<p>The issue doesnt seem that minor anymore. The  <span class="error">&#91;variant&#93;</span> is not searchable through CTRL+F. Also if u view the logs as plain text, you get no information of the variants. Having ~10MB of logs with >5 different variants makes it difficult to track the variant.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/dnusbaum.png" title="dnusbaum's avatar" /> dnusbaum:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=355846&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-355846">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I just released Pipeline Job Plugin 2.31, which shows the branch names for parallel branches whenever log output transitions from one branch to another in the classic UI. See the attached screenshot for an idea of what it looks like:</p>

<p> <span class="image-wrap" style=""><a id="45435_thumb" href="https://issues.jenkins.io/secure/attachment/45435/45435_JENKINS-54304.png" title="JENKINS-54304.png" file-preview-type="image" file-preview-id="45435" file-preview-title="JENKINS-54304.png"><img src="https://issues.jenkins.io/secure/thumbnail/45435/_thumb_45435.png" style="border: 0px solid black" role="presentation"/></a></span> </p>
</pre>
</details>

</ul>
</details>

<p>I just released Pipeline Job Plugin 2.31, which shows the branch names for parallel branches whenever log output transitions from one branch to another in the classic UI. See the attached screenshot for an idea of what it looks like:</p>

<p> <span class="image-wrap" style=""><a id="45435_thumb" href="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45435/<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/54304">JENKINS-54304</a>.png" title="<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/54304">JENKINS-54304</a>.png" file-preview-type="image" file-preview-id="45435" file-preview-title="<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/54304">JENKINS-54304</a>.png"><img src="https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/45435/<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/54304">JENKINS-54304</a>.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jglick.png" title="jglick's avatar" /> jglick:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=357033&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-357033">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<blockquote><p>Regarding branch names in the raw logs, I think it would be possible to add them back</p></blockquote>

<p>They were removed from the raw log intentionally; rendering of branch names is now done on the client side rather than being (redundantly) kept in the <tt>log</tt> file. Since the <tt>consoleText</tt> streams the raw log without any metadata, we do not see it. There could I suppose be some intermediate variant that processes the log in certain ways like this one but emits <tt>text/plain</tt>.</p>
</pre>
</details>

</ul>
</details>

<blockquote><p>Regarding branch names in the raw logs, I think it would be possible to add them back</p></blockquote>

<p>They were removed from the raw log intentionally; rendering of branch names is now done on the client side rather than being (redundantly) kept in the <tt>log</tt> file. Since the <tt>consoleText</tt> streams the raw log without any metadata, we do not see it. There could I suppose be some intermediate variant that processes the log in certain ways like this one but emits <tt>text/plain</tt>.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/macdrega.png" title="macdrega's avatar" /> macdrega:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=364255&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-364255">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Since this ticket is closed I have to ask: Is there already somewhere another ticket out there about the branch names not being searchable anymore? Doesn't work with Chrome or Firefox. 
And you clearly do not want to use IE or Edge with Jenkins - so I didn't even give them a shot.</p>

<p>In addition to that Chrome used to be the only browser being able to highlight all occurrences of a particular branch in the scroll bar - that way one could easily find an issue related to one branch only.</p>

<p>Especially for bigger pipelines with lots of branches we still need this.</p>

<p>The show/hide feature is quite nice. However what we would need is to be able to completely hide all branches (completely, all occurrences) except the chosen one. Only that would make the search obsolete. Is that / will that be possible?</p>
</pre>
</details>

</ul>
</details>

<p>Since this ticket is closed I have to ask: Is there already somewhere another ticket out there about the branch names not being searchable anymore? Doesn't work with Chrome or Firefox. <br/>
And you clearly do not want to use IE or Edge with Jenkins - so I didn't even give them a shot.</p>

<p>In addition to that Chrome used to be the only browser being able to highlight all occurrences of a particular branch in the scroll bar - that way one could easily find an issue related to one branch only.</p>

<p>Especially for bigger pipelines with lots of branches we still need this.</p>

<p>The show/hide feature is quite nice. However what we would need is to be able to completely hide all branches (completely, all occurrences) except the chosen one. Only that would make the search obsolete. Is that / will that be possible?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jglick.png" title="jglick's avatar" /> jglick:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=364256&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-364256">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<blockquote><p>Is there already somewhere another ticket out there</p></blockquote>

<p>If it is not <b>Link</b> ed here, then I would presume not.</p>

<blockquote><p>what we would need is to be able to completely hide all branches (completely, all occurrences) except the chosen one. Only that would make the search obsolete. Is that / will that be possible?</p></blockquote>

<p>It could be, using some CSS/JS manipulations, as <a href="https://github.com/jenkinsci/workflow-job-plugin/blob/846c3a359c04d3793dc0151ebd22f270a155949c/src/main/resources/org/jenkinsci/plugins/workflow/job/console/NewNodeConsoleNote/script.js#L18" class="external-link" target="_blank" rel="nofollow noopener">suggested here</a>. That would of course deserve its own (linked) issue.</p>
</pre>
</details>

</ul>
</details>

<blockquote><p>Is there already somewhere another ticket out there</p></blockquote>

<p>If it is not <b>Link</b> ed here, then I would presume not.</p>

<blockquote><p>what we would need is to be able to completely hide all branches (completely, all occurrences) except the chosen one. Only that would make the search obsolete. Is that / will that be possible?</p></blockquote>

<p>It could be, using some CSS/JS manipulations, as <a href="https://github.com/jenkinsci/workflow-job-plugin/blob/846c3a359c04d3793dc0151ebd22f270a155949c/src/main/resources/org/jenkinsci/plugins/workflow/job/console/NewNodeConsoleNote/script.js#L18" class="external-link" target="_blank" rel="nofollow noopener">suggested here</a>. That would of course deserve its own (linked) issue.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>sakshisood:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-54304?focusedId=364718&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-364718">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jglick" class="user-hover" rel="jglick">jglick</a> This change has made troubleshooting of the issues difficult. it's no longer clear which line belongs to which step. I have to scroll too many times to understand if i get some error to identify the error belongs to which step. Can this be reverted? If not, Is there any configuration change I can make to get the old behavior ?</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jglick" class="user-hover" rel="jglick">jglick</a> This change has made troubleshooting of the issues difficult. it's no longer clear which line belongs to which step. I have to scroll too many times to understand if i get some error to identify the error belongs to which step. Can this be reverted? If not, Is there any configuration change I can make to get the old behavior ?</p>
